tests/global_config_s390x.py

```diff
@@ -19,6 +19,7 @@
 
 rhel_os_matrix = generate_os_matrix_dict(os_name="rhel", supported_operating_systems=["rhel-9-5"])
 fedora_os_matrix = generate_os_matrix_dict(os_name="fedora", supported_operating_systems=["fedora-41"])
+centos_os_matrix = generate_os_matrix_dict(os_name="centos", supported_operating_systems=["centos-stream-9"])
```

Comment:
instead of duplicating it in all global_config_arch for s390x and x86 , have it in global_config.py . global config should take care of rest

Reply:
https://quay.io/repository/centos/centos?tab=tags but this shows arm64 images for centos

Reply:
@geetikakay we cannot assume we have image support for all architectures. it should be easy to see (under the relevant arch file what is supported). for multi-arch we will need to some more work)

Reply:
any effort should be tracked